package.json

```diff
@@ -8,59 +8,59 @@
 		"test": "./tools/scripts/use-make-instead"
 	},
 	"devDependencies": {
-		"@babel/core": "7.19.3",
-		"@babel/preset-env": "7.19.4",
+		"@babel/core": "7.20.5",
+		"@babel/preset-env": "7.20.2",
 		"@babel/preset-react": "7.18.6",
 		"@babel/preset-typescript": "7.18.6",
-		"@changesets/cli": "2.25.0",
-		"@emotion/babel-plugin": "11.10.2",
+		"@changesets/cli": "2.25.2",
+		"@emotion/babel-plugin": "11.10.5",
 		"@emotion/react": "11.0.0",
 		"@guardian/browserslist-config": "workspace:*",
 		"@guardian/eslint-config": "workspace:*",
 		"@guardian/eslint-config-typescript": "workspace:*",
 		"@guardian/prettier": "workspace:*",
 		"@manypkg/get-packages": "1.1.3",
-		"@nrwl/cli": "15.0.13",
-		"@nrwl/devkit": "15.0.13",
-		"@nrwl/eslint-plugin-nx": "15.0.13",
-		"@nrwl/jest": "15.0.13",
+		"@nrwl/cli": "15.2.1",
+		"@nrwl/devkit": "15.2.1",
+		"@nrwl/eslint-plugin-nx": "15.2.1",
+		"@nrwl/jest": "15.2.1",
 		"@nrwl/nx-cloud": "latest",
-		"@nrwl/workspace": "15.0.13",
-		"@storybook/addon-a11y": "6.5.12",
-		"@storybook/addon-docs": "6.5.12",
-		"@storybook/addon-essentials": "6.5.12",
-		"@storybook/addon-links": "6.5.12",
+		"@nrwl/workspace": "15.2.1",
+		"@storybook/addon-a11y": "6.5.13",
+		"@storybook/addon-docs": "6.5.13",
+		"@storybook/addon-essentials": "6.5.13",
+		"@storybook/addon-links": "6.5.13",
 		"@storybook/addons": "^6.5.13",
-		"@storybook/builder-webpack5": "6.5.12",
-		"@storybook/manager-webpack5": "6.5.12",
-		"@storybook/react": "6.5.12",
+		"@storybook/builder-webpack5": "6.5.13",
+		"@storybook/manager-webpack5": "6.5.13",
+		"@storybook/react": "6.5.13",
 		"@storybook/storybook-deployer": "2.8.16",
 		"@storybook/theming": "^6.5.13",
 		"@swc-node/register": "1.5.4",
-		"@swc/core": "1.3.8",
-		"@types/babel__core": "7.1.19",
-		"@types/jest": "29.1.2",
+		"@swc/core": "1.3.20",
+		"@types/babel__core": "7.1.20",
+		"@types/jest": "29.2.3",
 		"@types/node": "18.11.9",
 		"@types/react": "17.0.1",
-		"@typescript-eslint/parser": "5.42.0",
-		"babel-loader": "8.2.5",
-		"eslint": "8.25.0",
-		"husky": "8.0.1",
-		"jest": "29.2.0",
-		"jest-environment-jsdom": "29.2.0",
-		"lint-staged": "13.0.3",
-		"nx": "15.0.13",
-		"prettier": "2.7.1",
+		"@typescript-eslint/parser": "5.44.0",
+		"babel-loader": "9.1.0",
```

Comment:
do we know if this will work with other packages on older versions?

Reply:
no we dont' - i'll revert this. i'd like to discuss a process for how we could work this out though as it seems really unclear to me. I spose the package upgrade policy usually is "major upgrades only when we need 'em" as i understand it